loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -686,8 +687,30 @@ export class GLTFLoader implements IGLTFLoader {
         this.logClose();
 
         return Promise.all(promises).then(() => {
+            const min = TmpVectors.Vector3[0], max = TmpVectors.Vector3[1];
             this._forEachPrimitive(node, (babylonMesh) => {
-                babylonMesh.refreshBoundingInfo(true);
+                let recomputeBoundingInfo = true;
+                if (!this.parent.alwaysComputeBoundingBox && !babylonMesh.skeleton) {
+                    const tmp = GLTFLoader.GetTmpMetadata(babylonMesh);
+                    const mmin: [number, number, number] = tmp.positionMin, mmax: [number, number, number] = tmp.positionMax;
+                    if (mmin !== undefined && mmax !== undefined) {
+                        recomputeBoundingInfo = false;
+                        min.copyFromFloats(...mmin);
+                        max.copyFromFloats(...mmax);
+                        babylonMesh.getBoundingInfo().reConstruct(min, max);
+                        if (babylonMesh.subMeshes) {
```

Comment:
The sub meshes have the same min/max as the parent? That seems a bit weird.

Reply:
You told me that no sub meshes were created by the GLTF loader, so I think there is a single sub mesh and this sub mesh == main mesh?

Reply:
I could remove the code updating the sub meshes altogether as if there is a single sub mesh == main mesh, it will end up updating the same mesh twice...

Instead, I could add a comment saying that because GLTF does not create additional sub meshes, there's no need to update sub meshes bounding info?